1-exercises/F-array-forEach/exercise.js

```diff
@@ -9,6 +9,18 @@
 
 let arr = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15];
 
+arr.forEach(function (num){
+ if (num % 5 === 0 || num % 3 === 0) {
+    console.log("FizzBuzz");
```

Comment:
Not quite right. I think you meant "&&" instead of "||"
```javascript
if (num % 5 === 0 && num % 3 === 0) 
```
I have seen the Fizz Buzz exercise used many times in interviews, and you'd be surprised how many people get it wrong, even senior developers. They mainly get it wrong because they don't play close attention to the rules. Just for fun, this is what my version looked like:
```javascript
arr.forEach((n) => {
  let message = "";
  if (n % 3 === 0) message += "Fizz";
  if (n % 5 === 0) message += "Buzz";
  if (message === "") {
    console.log(n);
  } else {
    console.log(message);
  }
});
```
I strongly recommend that you memorise the Fizz Buzz implementation to avoid getting it wrong if you're asked to do it in an interview!